torch_geometric/transforms/__init__.py

```diff
@@ -84,6 +86,9 @@
     'GridSampling',
     'GCNNorm',
     'AddTrainValTestMask',
+    'Mesh',
```

Comment:
I don't think `torch_geometric.transforms` is the best place for these files. Maybe out-source to `*.data`?

Reply:
Do you mean torch_geometric.data?

Reply:
Yes :)